inc/central.class.php

```diff
@@ -184,7 +184,7 @@ static function showMyView() {
       }
 
       if ($DB->isSlave()
-          && !$DB->first_connection) {
+          && !$DB->isFirstConnection()) {
```

Comment:
I think this condition can be dropped. If you look at the code (mainly `CommonDBTM`), write operations are not done nor permitted when `$DB->isSlave()`.